user/rime-data/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "rime-data"
+pkgver = "0.20250123"
+pkgrel = 0
+_plum_commit = "4c28f11f451facef809b380502874a48ba964ddb"  # r126.20240417
+_subrepo_commits = {
```

Comment:
you can get rid of this whole array, and instead make a normal `sources` array; use `source_paths` to declare the layout of the extracted items